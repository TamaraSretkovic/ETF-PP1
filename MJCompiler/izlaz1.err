Exception in thread "main" java.lang.Error: Error: could not match input
	at rs.ac.bg.etf.pp1.Yylex.zzScanError(Yylex.java:530)
	at rs.ac.bg.etf.pp1.Yylex.next_token(Yylex.java:898)
	at rs.ac.bg.etf.pp1.MJParser.scan(MJParser.java:637)
	at java_cup.runtime.lr_parser.parse(lr_parser.java:578)
	at rs.ac.bg.etf.pp1.MJParser.main(MJParser.java:673)
