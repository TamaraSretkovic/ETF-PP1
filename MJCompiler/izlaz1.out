INFO  2013-08-17 03:35:40,956 [main][MJParser.java:667] : Compiling source file: /Users/ntrifunovic/Dropbox/IR4PP1/Projekat/MJCompiler/test/program1.mj
INFO  2013-08-17 03:35:40,987 [main][MJParser.java:639] : #2 program
INFO  2013-08-17 03:35:40,988 [main][MJParser.java:639] : #44 ProgramC
