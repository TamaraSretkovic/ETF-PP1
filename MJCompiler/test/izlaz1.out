INFO  2013-08-17 08:10:47,399 [main][MJParser.java:668] : Compiling source file: /Users/ntrifunovic/Dropbox/IR4PP1/Projekat/MJCompiler/test/program1.mj
INFO  2013-08-17 08:10:47,416 [main][MJParser.java:640] : #2 program
INFO  2013-08-17 08:10:47,416 [main][MJParser.java:640] : #44 Test1
INFO  2013-08-17 08:10:47,416 [main][MJParser.java:640] : #44 int
INFO  2013-08-17 08:10:47,417 [main][MJParser.java:2746] : Pocetak programa Test1 na liniji 1
INFO  2013-08-17 08:10:47,419 [main][MJParser.java:640] : #44 x
INFO  2013-08-17 08:10:47,419 [main][MJParser.java:640] : #44 Greska
INFO  2013-08-17 08:10:47,419 [main][MJParser.java:640] : #38 ;
INFO  2013-08-17 08:10:47,420 [main][MJParser.java:640] : #3 const
INFO  2013-08-17 08:10:47,420 [main][MJParser.java:640] : #44 int
INFO  2013-08-17 08:10:47,420 [main][MJParser.java:640] : #44 x
DEBUG 2013-08-17 08:10:47,421 [main][MJParser.java:2519] : Uspesan oporavak od greske na liniji 3 pri definisanju promenjive
INFO  2013-08-17 08:10:47,421 [main][MJParser.java:640] : #16 =
INFO  2013-08-17 08:10:47,421 [main][MJParser.java:640] : #38 ;
INFO  2013-08-17 08:10:47,423 [main][MJParser.java:640] : #34 {
INFO  2013-08-17 08:10:47,424 [main][MJParser.java:640] : #7 void
INFO  2013-08-17 08:10:47,424 [main][MJParser.java:640] : #44 main
DEBUG 2013-08-17 08:10:47,424 [main][MJParser.java:2677] : Uspesan oporavak od greske na liniji 5 pri definisanju konstante
INFO  2013-08-17 08:10:47,424 [main][MJParser.java:640] : #39 (
INFO  2013-08-17 08:10:47,425 [main][MJParser.java:640] : #40 )
INFO  2013-08-17 08:10:47,425 [main][MJParser.java:640] : #34 {
INFO  2013-08-17 08:10:47,425 [main][MJParser.java:640] : #35 }
INFO  2013-08-17 08:10:47,425 [main][MJParser.java:640] : #35 }
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2780] : Broj globalnih prostih promenjivih:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2781] : Broj globalnih nizova:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2782] : Broj globalnih konstanti:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2783] : Broj unutrasnjih klasa:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2784] : Broj iskaza u main-u:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2785] : Broj poziva metoda u main-u:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2786] : Broj blokova naredbi:0
INFO  2013-08-17 08:10:47,426 [main][MJParser.java:2787] : Broj funkcija glavnog programa:1
INFO  2013-08-17 08:10:47,427 [main][MJParser.java:2788] : Broj funkcija unutrasnjih klasa:0
INFO  2013-08-17 08:10:47,427 [main][MJParser.java:2789] : Broj naredbi kreiranja instanci unutrasnjih klasa:0
INFO  2013-08-17 08:10:47,427 [main][MJParser.java:2790] : Broj pristupa clanovima natklase preko kljucne reci super:0
=====================SYMBOL TABLE DUMP=========================
Type int: int, -1, -1 
Type char: char, -1, -1 
Con eol: char, 10, 0 
Con null: Class [], 0, 0 
Meth chr: char, 0, 1 
   Var i: int, 0, 1 

Meth ord: int, 0, 1 
   Var ch: char, 0, 1 

Meth len: int, 0, 1 
   Var arr: Arr of notype, 0, 1 

Type string: , -1, -1 
Type bool: , -1, -1 
Prog Test1: notype, 0, 1 
   Meth main: notype, 0, 0 



